charts/argocd-apps/values-integration.yaml

```diff
@@ -833,3 +833,70 @@ applications:
           secretKeyRef:
             name: smartanswers-zendesk-client
             key: password
+- name: feedback
+  helmValues:
+    appImage:
+      repository: 172025368201.dkr.ecr.eu-west-1.amazonaws.com/feedback
+      tag: latest  # To be edited by automation (not yet implemented).
+    replicaCount: 1
+    extraEnv:
+      - name: SECRET_KEY_BASE
+        valueFrom:
+          secretKeyRef:
+            name: rails-secret-key-base
+            key: SECRET_KEY_BASE
+      - name: SUPPORT_BEARER_TOKEN
+        valueFrom:
+          secretKeyRef:
+            name: signon-token-support-bearer-feedback
+            key: bearer_token  
+      - name: SUPPORT_API_BEARER_TOKEN
+        valueFrom: 
+          secretKeyRef:
+            name: signon-token-support-api-bearer-token-feedback
+            key: bearer_token
+      - name: PUBLISHING_API_BEARER_TOKEN
+        valueFrom:
+          secretKeyRef:
+            name: signon-token-publishing-api-bearer-feedback
+            key: bearer_token
+      - name: ASSISTED_DIGITAL_GOOGLE_SPREADSHEET_KEY
+        valueFrom: 
+          secretKeyRef:
+            name: assisted-digital-google-spreadsheet-key-feedback
+            key: bearer_token
+      - name: GOOGLE_CLIENT_EMAIL
+        valueFrom: 
+          secretKeyRef:
+            name: google-client-email-feedback
+            key: bearer_token
+      - name: GOOGLE_PRIVATE_KEY
+        valueFrom: 
+          secretKeyRef:
+            name: google-private-key-feedback
+            key: bearer_token
+      - name: GOVUK_NOTIFY_API_KEY
+        valueFrom: 
+          secretKeyRef:
+            name: govuk-notify-api-key-feedback
+            key: api_key
+       - name: GOVUK_NOTIFY_SURVEY_SIGNUP_TEMPLATE_ID
```

Comment:
Some of these don't need to be (and therefore shouldn't be) secrets:

* [GOVUK_NOTIFY_SURVEY_SIGNUP_TEMPLATE_ID](https://github.com/alphagov/govuk-puppet/search?q=GOVUK_NOTIFY_SURVEY_SIGNUP_TEMPLATE_ID)
* [GOVUK_NOTIFY_SURVEY_SIGNUP_REPLY_TO_ID](https://github.com/alphagov/govuk-puppet/search?q=GOVUK_NOTIFY_SURVEY_SIGNUP_REPLY_TO_ID)
* [GOVUK_NOTIFY_ACCESSIBLE_FORMAT_REQUEST_TEMPLATE_ID](https://github.com/alphagov/govuk-puppet/search?q=GOVUK_NOTIFY_ACCESSIBLE_FORMAT_REQUEST_TEMPLATE_ID)
* [GOVUK_NOTIFY_ACCESSIBLE_FORMAT_REQUEST_REPLY_TO_ID](https://github.com/alphagov/govuk-puppet/search?q=GOVUK_NOTIFY_ACCESSIBLE_FORMAT_REQUEST_REPLY_TO_ID)

Reply:
The reason for this is that secrets are costly (in terms of human effort) to maintain, so we definitely don't want to be storing things as secrets that we don't really need to.

Reply:
Thanks Chris - I'll remove these now.

Reply:
Identifier templates within GOV.UK Notify have been removed.